lib/shoulda/matchers/active_model/numericality_matchers/comparison_matcher.rb

```diff
@@ -77,7 +83,7 @@ def all_bounds_correct?
 
           def failing_submatchers
             submatchers_and_results.
-              reject { |x| x[:matched] }.
+              select { |x| !x[:matched] }.
```

Comment:
I know I've asked about this before but which cop controls this? This seems like an unnecessary check.

Reply:
This was reverted to its original state.
`Style/InverseMethods` controls this behaviour.

Reply:
Yeah, sorry for the bluntness. I just think it may lead to churn.